apps/docs/src/app/platform/documentation/platform-documentation.component.ts

```diff
@@ -30,6 +30,7 @@ export class PlatformDocumentationComponent {
         { url: 'platform/radio-group', name: 'Radio Button Group' },
         { url: 'platform/menu-button', name: 'Menu Button' },
         { url: 'platform/split-menu-button', name: 'Split Menu Button' },
+        { url: 'platform/info-label', name: 'Info Label' },
```

Comment:
can we reorder those in alphabetical order?

Reply:
done